src/renderers/webgl/WebGLState.js

```diff
@@ -677,6 +677,15 @@ function WebGLState( gl, extensions, utils, capabilities ) {
 		depthBuffer.setMask( material.depthWrite );
 		colorBuffer.setMask( material.colorWrite );
 
+		var stencilWrite = material.stencilWrite;
+		stencilBuffer.setTest( stencilWrite );
```

Comment:
It seems that this has broken the Masking examples in r.107. It is overwriting the stencil test independently set by the app.

A quick fix is to comment the line out.

/ping @gkjohnson

Reply:
E.g.: https://threejs.org/examples/webgl_postprocessing_masking

Reply:
Instead of just commenting out code and I vote to revert the PR for now. It can be merged again if existing code does not break.

Reply:
Good catch -- I didn't realize other code was using the stencil state considering it's not documented.

I've made two PRs: one that comments out the code as @WestLangley suggested (#17137) and one that fixes MaskPass (#17136) so it works with the new stencil materials.

My concern with just commenting out the code is that the documentation will still reflect the new stencil fields and depending on how long we expect this to be removed (until 108?) it could be confusing.

Reply:
>A quick fix is to comment the line out.

The suggestion to comment the line out was only intended to verify the offending line of code. I would not consider that as a hot fix.